app/containers/wallet/WalletSummaryPage.js

```diff
@@ -187,6 +190,35 @@ export default class WalletSummaryPage extends Component<InjectedOrGenerated<Gen
 
     const notification = this._getThisPageActiveNotification();
 
+    let exportDialog;
+
+    const delegationStore = this.generated.stores.delegation;
+    const delegationRequests = delegationStore.getDelegationRequests(publicDeriver);
+
+    if(!this.readyToExportHistory({ delegationRequests, publicDeriver })){
```

Comment:
Usually we have spaces between ifs and braces
ex
```
if (foo) {
} else {
}
```

I think it should be a lint warning

Reply:
weird that eslint isn't complaining about it, will check that out